packages/components/form/src/form-item.vue

```diff
@@ -52,7 +52,7 @@ import {
 } from 'vue'
 import AsyncValidator from 'async-validator'
 import { clone } from 'lodash-unified'
-import { refDebounced } from '@vueuse/core'
+import { refDebounced } from '@vueuse/shared'
```

Comment:
Importing should now be correct. https://vueuse.org/shared/refDebounced/#refdebounced

Reply:
@btea I think the documentation is not correct at this point. But some build tools still find the correct path so not everyone is experiencing this issue.
Please take a look at this:
https://github.com/vueuse/vueuse/issues/1688#issuecomment-1154232909

For me it worked using @rspack/cli in mode development but as soon as i made a production build it removed the /shared module (because of treeshaking?) and there was an error in the console.

Changing it to /shared fixed the issue. If you look inside the library the function actually lives in the shared module.
https://github.com/vueuse/vueuse/blob/main/packages/shared/refDebounced/index.ts

Reply:
So maybe it's a problem with rspack ? 🤔
https://github.com/vueuse/vueuse/pull/1940#issuecomment-1186536951

Reply:
@btea you're right, it was a bug with rspack. it was fixed in version `0.2.8` https://github.com/web-infra-dev/rspack/releases/tag/0.2.8